internal/service/cloudbackupschedule/resource_cloud_backup_schedule.go

```diff
@@ -643,7 +643,7 @@ func expandCopySetting(tfMap map[string]any) *admin.DiskBackupCopySetting {
 
 func expandCopySettings(tfList []any) *[]admin.DiskBackupCopySetting {
 	if len(tfList) == 0 {
-		return nil
+		return &[]admin.DiskBackupCopySetting{}
 	}
 
 	var copySettings []admin.DiskBackupCopySetting
```

Comment:
you can define the var with:
 copySettings := make([]admin.DiskBackupCopySetting,0)

and forget about the first if statement

Reply:
Thanks. Updated in f9d2e6964d64617b60dc4145853fe9c329533b68